lib/widgets/loading_button.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flashcards/features/auth/models/auth_exception.dart';
+import 'package:flashcards/helpers/dialog_handler.dart';
+import 'package:flutter/material.dart';
+
+class LoadingButton extends StatefulWidget {
+  LoadingButton({
+    super.key,
+    required this.text,
+    required this.onPressed,
+  });
+
+  final String text;
+  final Future<void> Function() onPressed;
+
+  @override
+  State<LoadingButton> createState() => _LoadingButtonState();
+}
+
+class _LoadingButtonState extends State<LoadingButton> {
+  bool isLoading = false;
```

Comment:
Why is here this variable? You are just setting it, but I don't see any other use

Reply:
True, It was used previously but I removed the functionality of the isLoading -> Removed